all-actors/src/main/java/org/sunbird/ActorOperations.java

```diff
@@ -9,6 +9,7 @@
     ADD("add"),
     VALIDATE("validate"),
     DOWNLOAD("download"),
+    DOWNLOADV2("downloadV2"),
```

Comment:
Can we enhance the same endpoint to accept both attributes - certUrl and pdfUrl? We can have minimal changes in this last min.